content/v2.0/reference/flux/stdlib/built-in/transformations/filter.md

```diff
@@ -52,15 +52,17 @@ Defaults to `drop`.
 _**Data type:** String_
 
 ##### drop
-Tables left without rows are dropped.
+Tables without rows are dropped.
 
 ##### keep
-Tables left without rows are output to the next transformation.
+Tables without rows are output to the next transformation.
 
 {{% warn %}}
-Keeping empty tables with your first `filter()` function can have severe performance costs.
+Keeping empty tables with your first `filter()` function can have severe performance
+costs since it retains empty tables from your entire data set.
 For higher performance, use your first `filter()` function to do basic filtering,
-then keep empty tables on subsequent `filter()` calls.
+then keep empty tables on subsequent `filter()` calls with smaller data sets.
```

Comment:
would folks do this, for example, to accurately aggregate data? If empty data points aren't included in the output, could you count the total? other use cases?

Reply:
That's one of the use cases, yes. This is covered in the `count()` doc.